DangerSwap/DangerSwap/Services/Scrapper/ScrapperService.cs

```diff
@@ -42,8 +42,8 @@ public void RunScrappers()
                 // Crypto
                 RunScript(_executionPaths.Item1, true);
                 // Fiat
```

Comment:
42 and 44 row, comments will be redundant after refactoring _executionPaths. Code will be self-documented